src/WalkingModule/app/robots/ergoCubSN000/dcm_walking/common/transformServer.ini

```diff
@@ -0,0 +1,9 @@
+# This configuration file is useful to start the frameTransformServer
+# device within the walking, or to run it externally via
+# yarpdev --from dcm_walking/common/transformServer.ini
```

Comment:
Was yarpdev deprecated?

Reply:
I don't think so. Looking at the code it does not seem to be about to be deprecated